content/en/integrations/_index.md

```diff
@@ -7,6 +7,20 @@ aliases:
     - /integrations/tcp_queue_length/
     - /integrations/snyk/
 description: Gather data from all of your systems, apps, & services
+cascade:
```

Comment:
this will not scale well as a solution for promoting integrations pages, but should hold us over in the interim.   more info https://gohugo.io/content-management/front-matter/#target-specific-pages